azure-pipelines.yml

```diff
@@ -33,11 +33,21 @@ jobs:
       sudo chmod +x /usr/local/bin/bazel
     displayName: Install Bazelisk
 
+  - task: InstallSSHKey@0
+    inputs:
+      hostName: "github.com"
+      knownHostEntry: "github.com,192.30.255.113 ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=="
```

Comment:
Is this something that CI needs to do? Can't it be a manual step?

Reply:
Yes, https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/install-ssh-key?view=azure-devops

Reply:
Can you add some comments?

Reply:
What comment in your mind? This is a normal required field in azp config, nothing special to comment. We're not linking to standard docs in circleci config like doing `add_ssh_keys`? Or do you want this in variables and not committed in a file?